docs/source/getting_started/index.rst

```diff
@@ -1,24 +1,113 @@
+
+.. code:: python
+
+    from dowhy import CausalModel
+    import dowhy.datasets
+
+    # Generate some sample data
+    data = dowhy.datasets.linear_dataset(
+        beta=10,
+        num_common_causes=5,
+        num_instruments=2,
+        num_samples=10000)
+
+    # Step 1: Create a causal model from the data and given graph.
+    model = CausalModel(
+        data=data["df"],
+        treatment=data["treatment_name"],
+        outcome=data["outcome_name"],
+        graph=data["gml_graph"])
+
+    # Step 2: Identify causal effect and return target estimands
+    identified_estimand = model.identify_effect()
+
+    # Step 3: Estimate the target estimand using a statistical method.
+    estimate = model.estimate_effect(identified_estimand,
+                                     method_name="backdoor.propensity_score_matching")
+
+    # Step 4: Refute the obtained estimate using multiple robustness checks.
+    refute_results = model.refute_estimate(identified_estimand, estimate,
+                                           method_name="random_common_cause")
+
+No idea what this means? Then the best place to learn more about this is the user guide's :doc:`.
+./user_guide/effect_inference/index` chapter. Alternatively, learn about its basic features in
+:doc:`../example_notebooks/dowhy_simple_example`.
+
+
+Graphical causal model-based inference
+---------------------------------------
+
+For features like root cause analysis, structural analysis and similar, DoWhy uses graphical causal models. The
+language of graphical causal models again offers a variety of causal questions that can be answered. DoWhy's API to
+answer these causal questions follows a simple 3-step recipe as follows:
+
+.. code:: python
+
+    import networkx as nx, numpy as np, pandas as pd
+    from dowhy import gcm
+
+    # Let's generate some "normal" data we assume we're given from our problem domain:
+    X = np.random.normal(loc=0, scale=1, size=1000)
+    Y = 2 * X + np.random.normal(loc=0, scale=1, size=1000)
+    Z = 3 * Y + np.random.normal(loc=0, scale=1, size=1000)
+    data = pd.DataFrame(dict(X=X, Y=Y, Z=Z))
+
+    # Step 1: Model our system:
+    causal_model = gcm.StructuralCausalModel(nx.DiGraph([('X', 'Y'), ('Y', 'Z')]))
+    gcm.auto.assign_causal_mechanisms(causal_model, data)
+
+    # Step 2: Train our causal model with the data from above:
+    gcm.fit(causal_model, data)
+
+    # Step 3: Perform a causal analysis. E.g. we have an:
+    anomalous_record = pd.DataFrame(dict(X=[.7], Y=[100.0], Z=[303.0]))
+    # ... and would like to answer the question:
+    # "Which node is the root cause of the anomaly in Z?":
+    anomaly_attribution = gcm.attribute_anomalies(causal_model, "Z", anomalous_record)
+
+Again, if this doesn't entirely make sense, yet, we recommend starting with
+:doc:`../user_guide/gcm_based_inference/index` in
+the user guide or check out :doc:`../example_notebooks/gcm_basic_example`.
+
+
+Further resources
```

Comment:
given the new structure, should the further resources go into the user guide section where we talk about causality?

Reply:
Yes, maybe. Have also already thought about the best place for this section. If it's okay, I would also like to post-pone this change until we have reworked the User Guide. Hopefully, that will give us a better understanding where this will fit.

Reply:
Sounds good!